### ISSUE TITLE
Unable to intract with InstallShield Wizard windows installer

### ISSUE THREAD

**Issue by sainivarun** [microsoft/WinAppDriver#1236]
WAD UIRecorder & WinAppDriver are unable to interact with InstallShield Wizard windows installer. Is it supported or not?
Contributors, please help..  @timotiusmargo

**Comment by pradeipp** [comment]
Did you try Inspect.exe?

**Comment by sainivarun** [comment]
@pradeipp Yes, tried Inspect.exe also. It gave me Automation ID/Name of the elements which resolved my issue with element inspection but further WinAppDriver is not identifying these locators while debugging test.

**Comment by pradeipp** [comment]
Hi @sainivarun were you able to resolve it? Winappdriver doesn't always detect all elements so you'd want to try different locator strategies like name, classname, or xpath.. try which one works.. it's a bit of a hit or miss sometimes.

**Comment by iamvkundra** [comment]
@pradeipp even i tried to automate the application which is packed with installshield. Can't get any controls of that application.

**Comment by Shakevg** [comment]
@iamvkundra Try to look on XML with window content and check (sometimes it can be different with inspect ): Session.PageSource;

**Comment by ereminmax** [comment]
I kind of have the similar issue , instead in my case there s a problem with detection of elements of the program which was started as a process directly from the test itself. Maybe issue is linked with windows security which prevents from interacting with the new window.

**Comment by shoaibmansoor** [comment]
@ereminmax I also faced the same issue. 
Starting Winappdriver Ui Recorder and Appium with admin rights worked for me.

**Comment by sainivarun** [comment]
Hi @pradeipp, I tried as above suggested but it didn't work.
